package.json

```diff
@@ -81,6 +81,7 @@
     "std-env": "^3.7.0",
     "terminate": "^2.6.1",
     "ufo": "^1.4.0",
+    "unwasm": "^0.3.7",
```

Comment:
We might upgrade to 0.4 and more minors while iterating over experimental feature. 

This dependency can increase dependency hoisting problems. Do we really need the `unwasm` types to be exposed to end users?

Reply:
maybe it isn't - I just sort of assumed from the name the type needed to flow through to end users.

but if it's purely internal for this module it should certainly be a dev dep...

Maybe another option could be setting it to `NitroOptions['wasm']` if you think that's safer?

Reply:
`NitroOptions['wasm']` seems a _better_ idea to avoid adding dep

Reply:
great. then it can flow through...